flink-end-to-end-tests/test-scripts/common_mesos_docker.sh

```diff
@@ -63,10 +67,9 @@ function start_flink_cluster_with_mesos() {
 
 function build_image() {
     echo "Building Mesos Docker container"
-    if ! retry_times $IMAGE_BUILD_RETRIES 0 docker build -f $END_TO_END_DIR/test-scripts/docker-mesos-cluster/Dockerfile \
+    if ! docker build -f $END_TO_END_DIR/test-scripts/docker-mesos-cluster/Dockerfile \
         -t flink/docker-mesos-cluster:latest \
         $END_TO_END_DIR/test-scripts/docker-mesos-cluster/; then
-        echo "ERROR: Could not build mesos image. Aborting..."
-        exit 1
+        return 1
```

Comment:
Shouldn't the `docker` command alone set a non-zero return value if something went wrong? So I guess the `if` block is not really needed.